src/routes/NestedTicketRoutes.js

```diff
@@ -14,7 +14,7 @@ const NestedTicketRoutes = ({ path }) => (
       <Header as="h5">Please select a ticket.</Header>
       <RecentTickets />
     </Route>
-    <Route path={`${path}/:ticketID`} component={Ticket} />
+    <Route path={`${path}/:ticketID/:height?/:width?`} component={Ticket} />
```

Comment:
Hmm... I don't think this needs to be in the URL, because we can retrieve it from the server response.

Reply:
I'm not quite sure how to request that from the server, because the current request to create a ticket doesn't send a height and width, I think?